packages/app/src/systems/Asset/services/assets.ts

```diff
@@ -109,6 +109,14 @@ export class AssetService {
       throw new Error('No assets to add');
     }
 
+    // validate that all of the names are defined
+    const someNameUndefined = assets.some((asset) => {
+      asset.name === undefined;
+    });
```

Comment:
I tested using the documentation add asset and the screen opens, I think you need to validate not only undefined but also is is not empty. It's better to also trim the the name to avoid empty spaces counting as a valid name. 

The following values should throw `'', ' ', '       ', undefined, null`

<img width="346" alt="Screenshot 2024-01-15 at 19 14 07" src="https://github.com/FuelLabs/fuels-wallet/assets/3941923/03f13371-5a97-462d-ad1f-a6290054b340">